### PR TITLE
BETA: Register kiwi.is-a.dev

### PR DESCRIPTION
**Pull request by sIumberdemon**
Added `kiwi.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/kiwi.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "imdakiki",
+        "email": "skullkid73@outlook.com"
+    },
+    "record": {
+        "CNAME": "https://kikisdevelopment.github.io/kikisite/"
```
Comment (Stef-00012) [inline domains/kiwi.json:7]: Invalid CNAME record, can not contain `https://` or `/`

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/kiwi.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "imdakiki",
+        "email": "skullkid73@outlook.com"
+    },
+    "record": {
+        "CNAME": "kikisdevelopment.github.io/kikisite/"
```
Comment (Stef-00012) [inline domains/kiwi.json:7]: it can't contain `/`, you should remove `/kikisite/`

**Review by Stef-DP — DISMISSED** [review]

File: domains/kiwi.json (hunk 2 above)
Comment (imdakiki) [reply domains/kiwi.json]: Done

Comment (imdakiki) [reply domains/kiwi.json]: What do I do now?

Comment (imdakiki) [reply domains/kiwi.json]: I updated the JSON

**Review by omsenjalia — DISMISSED** [review]
we dont allow website that provide services

**Review by omsenjalia — APPROVED** [review]

**Review by VaibhavSys — APPROVED** [review]

**Pull request merged**